store.go

```diff
@@ -3354,6 +3354,20 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 		storeOptions.GraphDriverName = os.Getenv("STORAGE_DRIVER")
 	}
 
+	if storeOptions.GraphDriverName == "" {
+		options := drivers.Options{
+			Root:    storeOptions.GraphRoot,
+			RunRoot: storeOptions.RunRoot,
+			UIDMaps: storeOptions.UIDMap,
+			GIDMaps: storeOptions.GIDMap,
+		}
+		name, err := drivers.GetDefaultDriverName(options)
+		if err != nil {
+			fmt.Print(err)
```

Comment:
Should the error be returned here, too?

Reply:
The errors above for parsing the UID Maps were just printing errors, so I was matching, but I am fine with returning an error.  Problem is ReloadConfigurationFile currently does not return errors, so this would need to be an API Breaking change.